spec/automation/unit/method_validation/rejected_spec.rb

```diff
@@ -0,0 +1,50 @@
+describe 'Quota rejected Validation' do
+  let(:admin) do
+    FactoryGirl.create(:user_with_email_and_group,
+                       name: 'admin',
+                       userid: 'admin')
+  end
+  let(:admin_approval) { FactoryGirl.create(:miq_approval, approver: admin) }
+  let(:ws) do
+    MiqAeEngine.instantiate("/System/Request/Call_Method?#{method}&"\
```

Comment:
I'm really not a fan of the line-continuator.  We don't have 80 character limits in ManageIQ, so please just keep these all as one big string.  It's much easier to read that way.